### PR TITLE
Updated the myPlace model and placeResult model

### PR DESCRIPTION
**Pull request by amandacost9**

### CODE REVIEW

**Comment by amandacost9** [comment]
The model results have to be named to match the JSON result names so I made sure they do. 
myPlace model will be database saved things AND placeResult will be anything from the Google API

File: src/app/models/my-place.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+export class MyPlace {
+    myPlaceId: number;
+    visited: boolean;
+    userId: number;
+    createdOn: string;
+    googlePlaceId: string;
+
```
Comment (kendall-mchenry) [inline src/app/models/my-place.ts]: Add: 
to top of file: import { User } from "./user";

userId?: User["userId"];
user?: User;

File: src/app/models/user.ts [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
-    email?: string;
-    password?: string;
-  
-    constructor(email?: string, password?: string) {
-      this.email = email;
-      this.password = password;
-    }
+  username?: string;
+  password?: string;
+
```
Comment (kendall-mchenry) [inline src/app/models/user.ts:13]: Build out user model

File: src/app/models/place-result.ts [hunk 3]
```diff
@@ -0,0 +1,39 @@
+export class PlaceResult {
+  placeId: string;
```
Comment (kendall-mchenry) [inline src/app/models/place-result.ts]: This needs to be place_id

File: src/app/models/place-result.ts [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+  name?: string;
+  type?: string[];
```
Comment (kendall-mchenry) [inline src/app/models/place-result.ts]: I think this needs to be types

File: src/app/models/my-place.ts (hunk 1 above)
Comment (amandacost9) [reply src/app/models/my-place.ts]: Done.

File: src/app/models/user.ts (hunk 2 above)
Comment (amandacost9) [reply src/app/models/user.ts:13]: Done

File: src/app/models/place-result.ts (hunk 4 above)
Comment (amandacost9) [reply src/app/models/place-result.ts]: done

File: src/app/models/place-result.ts (hunk 3 above)
Comment (amandacost9) [reply src/app/models/place-result.ts]: done

**Pull request merged**